src/implementations/twig/components/page-header-standardised/page-header-standardised.html.twig

```diff
@@ -0,0 +1,74 @@
+{% spaceless %}
+{#
+  Parameters:
+    - "title" (string) (default: '') Title of header
+    - "description" (string) (default: '') Description of header
+    - "meta" (string) (default: '') Meta of header
+    - "breadcrumb" (associative array) (default: '') Predefined structure compatible with EC Breadcrumb
+    - "extra_classes" (string) (default: '')
+    - "extra_attributes" (array) (default: []): format: [
+        {
+          "name" (string) (default: ''),
+          "value" (optional) (string)
+        ...
+      ],
+    - "_compliance_" (boolean) (default: false) Activates debug
+#}
+
+{# variables #}
+{% set _title = title|default('') %}
+{% set _meta = meta|default('') %}
+{% set _description = description|default('') %}
+{% set _compliance = _compliance_|default(false) %}
+{% set _css_class = 'ecl-page-header-standardised' %}
+{% set _extra_attributes = '' %}
+{% set _breadcrumb = breadcrumb|default({}) %}
+
+{# Internal logic - Process properties #}
+{% if extra_classes is defined and extra_classes is not empty %}
+  {% set _css_class = _css_class ~ ' ' ~ extra_classes %}
+{% endif %}
+
+{% if extra_attributes is defined and extra_attributes is not empty and extra_attributes is iterable %}
+  {% for attr in extra_attributes %}
+    {% if attr.value is defined %}
+      {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name ~ '="' ~ attr.value|e ~ '"' %}
+    {% else %}
+      {% set _extra_attributes = _extra_attributes ~ ' ' ~ attr.name %}
+    {% endif %}
+  {% endfor %}
+{% endif %}
+
+{# Print the result #}
+<div class="{{ _css_class }}" {{ _extra_attributes|raw }}>
+  <div class="ecl-container">
+  {% if _breadcrumb is not empty %}
+    {% include '@ecl/breadcrumb-standardised/breadcrumb-standardised.html.twig' with _breadcrumb|merge({
+      extra_classes: 'ecl-page-header-standardised__breadcrumb',
+      _compliance_: _compliance,
+      _compliance_inner_check_: true
+    }) only %}
+  {% endif %}
+  {% if _meta is not empty %}
+    <div class="ecl-page-header-standardised__meta">
+      <span class="ecl-u-type-uppercase">
```

Comment:
So....we just released in ECL v2 an updated version of the css for these components, where the meta is uppercased in the CSS and i see that we have it in the css you are adding in the vanilla component, so this <span> can now disappear, i write this here but you know it's the same in all the families, sorry, i could only notice this now that i worked on aligning the twig component ;)

Reply:
Nice catch! Updated on 3 page-headers ;)